web/views/job_view.ex

```diff
@@ -4,4 +4,29 @@ defmodule Community.JobView do
   def sanitize(text) do
     HtmlSanitizeEx.basic_html(text)
   end
+
+  def can_edit?(job, token) do
+    job.token == token
+  end
+
+  def trix_input(form, field_name, attrs \\ []) do
+    connected_trix_input(form, field_name) ++
+    trix_tag(field_name, attrs)
```

Comment:
Maybe indent this, or keep it on one line so it's not lost that you're appending these?